horace_core/sqw/PixelData/@PixelData/PixelData.m

```diff
@@ -236,10 +236,12 @@
         end
         % In memory construction
         if isa(arg, 'PixelData')  % TODO make sure this works with file-backed
-            if ~isempty(arg.file_path) && exist(arg.file_path, 'file')
+            if arg.is_file_backed_() && exist(arg.file_path, 'file')
                 % if the file exists we can create a file-backed instance
                 obj = PixelData(arg.file_path, arg.page_memory_size_);
                 obj.page_number_ = arg.page_number_;
+                obj.page_dirty_ = arg.page_dirty_;
+                arg.tmp_io_handler_.copy_tmp_folder(obj.object_id_);
```

Comment:
Would it be clearer as `arg.tmp_io_handler_.copy_folder(obj.object_id_)`? It's quite hard to read this link is snake case already and repetition of `_tmp` makes it harder -- the tmp_io_handler only does things with tmp_dirs and tmp_files.